### ISSUE TITLE
Autofill the current game assigned to the profile for the Steam App ID in the "Guided" tool editior

### ISSUE THREAD

**Issue by poperigby** [limo-app/limo#69]
The Protontricks runtime option has you input the Steam App ID of the game you want to run the tool under. I think it would be reasonable to assume that the game assigned to the current profile would be the one the user wants to use. If not, they can easily changed it.

**Comment by limo-app** [comment]
I agree. Currently Limo does not store Steam App IDs, so for existing games this would have to be extracted from the command used to run the game.
I will update the way games launch commands are input to work the same way it does for tools in a future update, allowing for a direct way to access App IDs.
